src/Generator.php

```diff
@@ -291,4 +296,55 @@ protected function error($string)
             echo $string . "\r\n";
         }
     }
+
+    /**
+     * Add all macroable classes which are not already loaded as an alias and have defined macros.
+     *
+     * @param Collection $aliases
+     */
+    protected function addMacroableClasses(Collection $aliases)
+    {
+        $macroable = $this->getMacroableClasses($aliases);
+
+        foreach ($macroable as $class) {
+            $reflection = new ReflectionClass($class);
+
+            if (!$reflection->getStaticProperties()['macros']) {
+                continue;
+            }
+
+            $aliases[] = new Alias($this->config, $class, $class, [], $this->interfaces);
+        }
+    }
+
+    /**
+     * Get all loaded macroable classes which are not loaded as an alias.
+     *
+     * @param Collection $aliases
+     * @return Collection
+     */
+    protected function getMacroableClasses(Collection $aliases)
+    {
+        return collect(get_declared_classes())
```

Comment:
> `collect`

Please don't use global helpers, it's a bad practice doing this in libraries. None of the helpers from `Illuminate\Support\helpers.php` is currently in use.

Using the collection of course is fine!